cli/command/image/remove.go

```diff
@@ -56,9 +57,13 @@ func runRemove(dockerCli command.Cli, opts removeOptions, images []string) error
 	}
 
 	var errs []string
+	var unresolvableErr = false
 	for _, img := range images {
 		dels, err := client.ImageRemove(ctx, img, options)
 		if err != nil {
+			if !apiclient.IsErrImageNotFound(err) {
```

Comment:
minor, but these object specific error checkers should be deprecated. Please use `IsErrNotFound()` instead. `IsErrImageNotfound()` just calls that directly anyway.